doc/languages-frameworks/python.section.md

```diff
@@ -655,6 +654,9 @@ Another difference is that `buildPythonPackage` by default prefixes the names of
 the packages with the version of the interpreter. Because this is irrelevant for
 applications, the prefix is omitted.
 
+When packaging a python application with `buildPythonApplication`, it should be
+invoked with `callPackage` and passed `python` or `pythonPackages`.
```

Comment:
One of the issues with the python packages is that there are a handful of historical bad examples floating around.  In this PR I'm making an effort to start noting some of the best practices, though there's more editing to be done for sure.

Reply:
invoked with -> called with

Reply:
Maybe show an example of a Python application?